static/app/views/settings/project/server-side-sampling/utils/index.tsx

```diff
@@ -8,7 +8,7 @@ import {defined} from 'sentry/utils';
 import {projectStatsToSampleRates} from './projectStatsToSampleRates';
 
 export const SERVER_SIDE_SAMPLING_DOC_LINK =
-  'https://docs.sentry.io/product/data-management-settings/server-side-sampling/';
+  'https://docs.sentry.io/product/data-management-settings/dynamic-sampling/';
```

Comment:
I tried to open this page and got 404... is this going to be updated soon?

Reply:
there is one [https://docs.sentry.io/product/sentry-basics/sampling/](https://docs.sentry.io/product/sentry-basics/sampling/)

Reply:
yeah, there needs to be docs PR merged in first 👍
we are renaming it there to dynamic sampling too - let me add it to the PR description